### PR TITLE
Simple Text Annotation app with Tailwind + Markdown

### PR DESCRIPTION
**Pull request by altryne**
Wanted to take fastHTML for a spin (as a former frontend dev) and submit a demo app for folks who are familiar with tailwind a lot as an example.

image

## Features

- View and navigate through text samples
- Annotate texts as correct or incorrect
- Add notes to each text sample
- Real-time updates without page reloads
- SQLite database integration
- Automatic loading of dummy data

### CODE REVIEW

File: annotate_text/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+python-fasthtml
+uvicorn>=0.29
+claudette
```
Comment (johnowhitaker) [inline annotate_text/requirements.txt:3]: I don't think this is being used right?

File: annotate_text/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# FastHTML Text Annotation App
+
+This project implements a simple text annotation application using FastHTML, showcasing dynamic updates, database integration, and a user-friendly interface for annotating text data.
+
+![Text Annotation App Screenshot](https://gist.github.com/user-attachments/assets/f28aff8c-3a55-439d-b605-d68c0b82c1b3)
+
```
Comment (johnowhitaker) [inline annotate_text/README.md:6]: "Contributed by: @yourusername" might be nice to add :)

File: annotate_text/requirements.txt (hunk 1 above)
Comment (altryne) [reply annotate_text/requirements.txt:3]: oh right, this is me copying from your previous requirements, removing

**Comment by johnowhitaker** [comment]
image
Dark mode looks a little strange, maybe force light mode if that's possible? It might be PicoCSS causing it, you could possible exclude that if you're mainly using tailwind.

**Review by altryne — COMMENTED** [review]
> image
Dark mode looks a little strange, maybe force light mode if that's possible? It might be PicoCSS causing it, you could possible exclude that if you're mainly using tailwind.  

Isn't pico included with the app by default? how can I try excluding?

**Comment by johnowhitaker** [comment]
https://picocss.com/docs/color-schemes < easiest way to turn off dark mode if it is Pico's fault is to set data-theme="light" (https://picocss.com/docs/color-schemes) - I used `bodykw={"data-theme":"light"}` in the fast_app args and it seems to work

**Comment by johnowhitaker** [comment]
For ref if you did want to remove it, don't use `picolink` in hdrs (`hdrs=(tlink, dlink, picolink...`) and set default_hdrs and pico args to False

**Comment by johnowhitaker** [comment]
Great, LGTM :+1:

**Pull request merged**